src/test/java/org/openstreetmap/atlas/checks/event/CheckFlagGeoJsonProcessorTest.java

```diff
@@ -92,7 +92,6 @@ private void processCompleteAndValidate(final int eventCount)
                 actualEventCount += found.getAsJsonArray("features").size();
             }
         }
-
```

Comment:
Whats the purpose of this change? To keep the PR footprint as small as possible, we should only update lines of code that are required by your PR